src/oc_erchef/apps/chef_objects/src/chef_regex.erl

```diff
@@ -74,6 +74,9 @@
 %% Validation regex for various names
 -define(HUMAN_NAME_REGEX, "[[:word:][:digit:]!'. -]+").
 
+%% Validation regex for various names that can be empty.
+-define(OPTIONAL_NAME_REGEX, "[[:word:][:digit:]!'. -]*").
```

Comment:
Perhaps we should change `HUMAN_NAME_REGEX` since first_name and last_name are also optional?  The only non-optional field is display_name, but my guess is we have previously allowed empty strings there as well.